### PR TITLE
Clean up kops build targets

### PR DESCRIPTION
**Pull request by olemarkus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -184,34 +177,40 @@ test:
 test-windows:
 	go test -v $(go list ./... | grep -v /nodeup/)
 
+
+.PHONY: kops
+kops: crossbuild-$(shell go env GOOS)-$(shell go env GOARCH)
```
Comment (hakman) [inline Makefile:182]: Maybe we should be more explicit and use `crossbuild-kops-...`, or just drop `crossbuild-` completely?

Comment (olemarkus) [reply Makefile:182]: I thought about it, but didn't want to change the naming of targets in this PR. But it is something we should look at.

Comment (hakman) [reply Makefile:182]: I think the only one that already exists is `crossbuild`, all the rest are new.

**Review by hakman — APPROVED** [review]

**Pull request merged**